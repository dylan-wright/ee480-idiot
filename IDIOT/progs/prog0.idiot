xor $u1, $u2
