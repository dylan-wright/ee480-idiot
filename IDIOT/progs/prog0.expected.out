Text:
//generated by AIK version 20070512
@0000
824a
//end
Data:
//generated by AIK version 20070512
//end
